data/gegno/gegno missions.txt

```diff
@@ -0,0 +1,103 @@
+			choice
+				`	"The Gegno?"`
+					goto gegno
+				`	"Are the ships I’ve seen these Gegno?"`
+					goto ships
+			label gegno
+			`	"Yes, the imperious Gegno. A juvenile race, they are, many decades younger than that of you humans. They do not find themselves in the struggle of leadership the same way as others do. Rather, they glorify their pursuit of guidance between one another to share a noble future."`
+			choice 
+				`	"What is this guidance they pursue?"`
+					goto guidance
+			label ships
+			`	"Indeed, the vessels of such stature represent the robust Gegno. Not in sorrow, but greatness do they pursue their future, bearing the fruit of sacrifice as a trophy to their name." The Quarg now shifts facial expressions in such a way to suggest a small hint of frustration."`
+			choice
+				`	"Why do they pursue in the name of sacrifice?"`
+					goto sacrifice
+			label guidance
+			`	The Quarg now shifts facial expressions in such a way to suggest a small hint of frustration. "The Gegno believe it to be necessary to sacrifice whatever must be, in order to imagine a future for themselves. Thus, they have separated into two different branches. One brandishes themselves as the honorable Vi- mightiest of warriors, toughest of kin. They are brutes that see to proof of strength being the one and only variable in determining their path. That of the Scin, the more technologically able, believe that only through progression of sciences and discovery, will the Gegno prosper."`
+			label sacrifice
+			`	"The Gegno believe it to be necessary to sacrifice whatever must be, in order to imagine a future for themselves. Thus, they have separated into two different branches. One brandishes themselves as the honorable Vi- mightiest of warriors, toughest of kin. They are brutes that see to proof of strength being the one and only variable in determining their path. That of the Scin, the more technologically able, believe that only through progression of sciences and discovery, will the Gegno prosper."`
+
+			choice
+				`"The Vi are right, the strength to survive is more important than anything else." `
+					goto vi
+				`"The Scin are right, technological advancement opens up so many possibilities."`
+					goto scin
+				`"Why do they not combine their efforts to better their future together?"`
+					goto together
+			label vi
+			`	"While it is characteristic, such a tremendous proposition is rather dull for a growing species. Be that as it may, you humans follow to a much lesser degree than that of the Vi."`
+			label scin
+			`	The Quarg now shifts facial expressions in such a way to suggest a small hint of disappointment. "The pursuit of technology leaves behind many values important to a species' growth."`
+			label together
+			`	"Regrettably, the Gegno chiefly do not have a drive to pursue their future, diluting themeselves far into their present. Segregation of followings was one of few ways to seek motivation beyond the fact."`
+			choice
```

Comment:
```suggestion
			label vi
			`	"While it is characteristic, such a tremendous proposition is rather dull for a growing species. Be that as it may, you humans follow to a much lesser degree than that of the Vi."`
				goto end
			label scin
			`	The Quarg now shifts facial expressions in such a way to suggest a small hint of disappointment. "The pursuit of technology leaves behind many values important to a species' growth."`
				goto end
			label together
			`	"Regrettably, the Gegno chiefly do not have a drive to pursue their future, diluting themeselves far into their present. Segregation of followings was one of few ways to seek motivation beyond the fact."`
			label end
			choice
```